app/assets/stylesheets/govuk_publishing_components/components/_layout-footer.scss

```diff
@@ -21,10 +21,43 @@
   margin-bottom: 20px;
 }
 
-// stylelint-disable declaration-no-important
 @include govuk-media-query($media-type: print) {
+  .govuk-footer {
+    background: none;
+    margin: 25mm 0 0;
```

Comment:
Was there a specific reason for this much margin? I'm mindful that the more margin there is, the more likely the footer will spill onto another page, which would waste paper.

Reply:
Yes, this was discussed with Nikin and we agreed that more space was needed between the last element of the page and the footer, otherwise it looks too cramped.